pkg/common/catalog/catalog_test.go

```diff
@@ -47,7 +48,7 @@ func (c *CatalogTestSuite) SetupTest() {
 	cat := &catalog{
 		pluginConfigs: PluginConfigMap{"NodeAttestor": {"join_token": HclPluginConfig{
 			PluginCmd:  "./attestor",
-			Enabled:    true,
```

Comment:
nitpick: seems a little heavy handed to import the Azure package just to make a `*bool`:

```
enabled := true
cat := &catalog{
    pluginConfigs: PluginConfigMap{"NodeAttestor": {"join_token": HclPluginConfig{
        PluginCmd: "./attestor",
        Enabled: &enabled,
...
```